simplifycommerce.php

```diff
@@ -450,7 +450,7 @@ public function processPayment()
 				'?id_cart='.(int)$this->context->cart->id.'&id_module='.(int)$this->id.'&id_order='.
 				(int)$this->currentOrder.'&key='.$this->context->customer->secure_key, '');
 		else
-			Tools::redirect(Link::getPageLink('order-confirmation.php', null, null,
+			Tools::redirect($this->context->link->getPagelink('order-confirmation.php', null, null,

```

Comment:
Can you do a text search across the codebase & see if we need to replace it elsewhere?